compute/metadata/README.md

```diff
@@ -11,5 +11,6 @@ These samples demonstrate interacting with the Compute Engine metadata service.
 These samples are used on the following documentation page:
 
 > https://cloud.google.com/compute/docs/metadata
```

Comment:
Could you update this link? It redirects to `https://cloud.google.com/compute/docs/storing-retrieving-metadata` which I think it still redirects to the link you want but let's update the original link as well.